dbg.h

```diff
@@ -463,6 +463,17 @@ class DebugOutput {
     return std::forward<T>(value);
   }
 
+  // Print statement for empty dbg() calls
+  void print() const {
+    std::stringstream output;
+    output << ansi(ANSI_DEBUG) << "[" << m_filepath << ":" << m_line << " ("
+           << m_function_name << ")] " << ansi(ANSI_RESET);
+    // Use expression as value, avoids types
+    output << ansi(ANSI_VALUE) << m_expression << ansi(ANSI_RESET);
+    output << std::endl;
+    std::cerr << output.str();
+  }
+
```

Comment:
I would really like to try to avoid this code duplication.

Couldn't we call `.print("dbg call reached")` and somehow leave the expression part empty instead?

Reply:
That commit has been removed.